src/LazyOperations/ResetMap.jl

```diff
@@ -314,7 +303,7 @@ function constraints_list(rm::ResetMap{N}) where {N<:Real}
         return res
     end
 
-    constraints = copy(constraints_list(LinearMap(get_A(rm), rm.X)))
+    constraints = copy(constraints_list(LinearMap(matrix(rm), rm.X)))
```

Comment:
```suggestion
    constraints = copy(constraints_list(LinearMap(matrix(rm), set(rm))))
```

Reply:
`rm.X` is old code that I did not touch. I will change it now. But while it is better to never do that, I think it is fine to use the field access instead of the getter function in the main file that defines the type. We do that all over this library.

Reply:
> I think it is fine to use the field access instead of the getter function in the main file that defines the type. 

Yes, certainly. I commented because I expected to see `set(..)` as there is `matrix(..)` in the same line.